app/src/main/java/com/codesoom/assignment/product/ui/dto/ProductResponseDto.java

```diff
@@ -0,0 +1,58 @@
+package com.codesoom.assignment.product.ui.dto;
+
+import com.codesoom.assignment.product.domain.Product;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+/**
+ * 상품의 정보를 담은 응답 DTO.
+ */
+@Getter
+@EqualsAndHashCode(of = "id")
+public class ProductResponseDto {
+
+    private Long id;
+
+    private String name;
+
+    private String maker;
+
+    private int price;
+
+    private String imageUrl;
+
+    /**
+     * 상품정보 응답 생성자.
+     *
+     * @param id       상품 식별자
+     * @param name     상품명
+     * @param maker    상품제조사
+     * @param price    상품가격
+     * @param imageUrl 상품이미지
+     */
+    @Builder
+    public ProductResponseDto(Long id, String name, String maker, int price, String imageUrl) {
+        this.id = id;
+        this.name = name;
+        this.maker = maker;
+        this.price = price;
+        this.imageUrl = imageUrl;
+    }
+
+    /**
+     * 상품 정보를 응답 DTO로 변경하는 정적 팩토리 메서드.
```

Comment:
정적 팩토리 메서드라는 것을 주석에서 밝힐 필요는 없습니다.
```suggestion
     * 상품 정보로 응답용 상품 정보 DTO를 생성해 리턴합니다.
```

Reply:
제가 주석에 지나친 정보를 담았네요!